packages/aragon-cli/src/commands/dao_cmds/acl_cmds/utils/params.js

```diff
@@ -0,0 +1,156 @@
+const { isString } = require('lodash')
```

Comment:
Someone get the fire trucks, cause this is hot 🔥:fire: :fire:

Haven't checked the correctness (we should certainly add more tests), but:

- We should scope this out into its own package
- We may be able to define a light DSL language for this and use something like [nearley](https://nearley.js.org) or [peg.js](https://pegjs.org) to parse it; we would most likely want to use this in a browser environment so we'll have to be conscious of the parser's size (and obviously handrolling it like now would be the most optimal)
- We should add a decoder to go from the param bytes, for use in UI

Reply:
Hahaha thanks! :smile: 
Definitely good ideas. I added them to my to-do list. I'm not super familiar with DSLs howver, I will have to read the subject a little bit more. But point 1 and 3 should be fairly straightforward to do.